server/src/infra/repositories/asset.repository.ts

```diff
@@ -805,9 +805,13 @@ export class AssetRepository implements IAssetRepository {
   }
 
   @GenerateSql({ params: [DummyValue.STRING, DummyValue.UUID, { numResults: 250 }] })
```

Comment:
You'd need to adapt this since there is now an array of user ids => `params: [DummyValue.STRING, [DummyValue.UUID], { numResults: 250 }]`

Reply:
Thanks, missed that!